go.mod

```diff
@@ -7,6 +7,9 @@ require (
 	github.com/mattn/go-colorable v0.0.9 // indirect
 	github.com/mattn/go-isatty v0.0.4 // indirect
 	github.com/pkg/errors v0.8.1
+	github.com/smartystreets/goconvey v1.6.4 // indirect
```

Comment:
Suspicious dependency. Seems like http2curl doesn't have this as a dependency. Could you elaborate what is the purpose of this?